### ISSUE TITLE
After Update (Contao 4.4.32 -> 4.4.53 and maybe this bundle??) body class not working anymore

### ISSUE THREAD

**Issue by bopsoid** [heimrichhannot/contao-hyphenator-bundle#3]
Hello,

after some tests from helpful members of the contao community, they found out, that after my update from Contao 4.4.32 -> 4.4.53 (I don't know, if there was also an update of the hyphenator bundle during this process) the body class (which can be set in page layout and page data itself) was not output to the frontend source code anymore.

See this issue here:
https://github.com/contao/contao/issues/2386

As those friendly Contao members pointed out to me, this might be a problem with the hyphenator bundle. Thus I'm posting this problem as an issue right here.

Thanks and many greetings,
Boris

**Comment by Defcon0** [comment]
Does it disappear after deactivating the hyphenator bundle in your case?

**Comment by Defcon0** [comment]
I can remeber having that problem when erroneously adding a div to the head section which caused the browser to intrepret it as the first element in the dom which isn't the head and therefor making it the body. In my case the body also was then placed somewhere else below.

**Comment by koertho** [comment]
@bopsoid Just installed the hyphenator bundle in a clean contao 4.4.53, everthing works as excepted. So the error is very likely is your template.

**Comment by bopsoid** [comment]
@Defcon0:

Yes, error is gone after uninstalling hyphenator bundle. No DIV in the header section.

@koertho:

Template was not changed, it's the original template "fe_page.html" shipped with Contao.
Another member of the Contao core team could reproduce the error, he had the same problem with disappearing body classes after installing hyphenator bundle. The error was immediately gone on my website after uninstalling the hyphenator bundle.

**Comment by koertho** [comment]
@bopsoid Could you please try in an clean contao install and report back? Maybe there is some configuration that leads to that problem.

**Comment by bopsoid** [comment]
Hello,

did a clean install and yes: no problems there. Funny enough, that the error was reproducible by a member of the core team as well. Maybe there are two modules installed that interfere with each other?

I'll now try to install one module after the other of those I'm using in the other website and see, in combination with which one the error might reappear.

Thanks so far, will send another comment when finding out more.
Boris

**Comment by bopsoid** [comment]
Ok, finally I found the conflicting modules:

Just after installing con4gis/maps  the class in the body was not set anymore (in combination with installed hyphenator module). Without hyphenator module it works again.

So there seems to be a conflict between those two modules.

Greetings,
Boris
